insights/parsers/ironic_conf.py

```diff
@@ -0,0 +1,7 @@
+from .. import parser, IniConfigFile
+from insights.specs import Specs
+
+
+@parser(Specs.ironic_conf)
+class IronicConf(IniConfigFile):
+    pass
```

Comment:
@smalleni please review the [documentation guidelines](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html) and add documentation to your new parser.